temba/msgs/models.py

```diff
@@ -880,8 +880,10 @@ def process_message(cls, msg):
             chatbase_not_handled = False
 
         # Sending data to Chatbase API
-        (chatbase_api_key, chatbase_version) = msg.org.get_chatbase_credentials()
-        if chatbase_api_key and msg.channel:
+        (chatbase_api_key, chatbase_version) = msg.org.get_chatbase_credentials() if not msg.contact.is_test \
+            else (None, None)
+
+        if chatbase_api_key:
```

Comment:
Seems simpler to write this as ...

```python
if not msg.contact.is_test:
  (chatbase_api_key, chatbase_version) = msg.org.get_chatbase_credentials()
  if chatbase_api_key:
    ...
```

no?

Reply:
Yes, I'll change